### PR TITLE
add dist

### PR DESCRIPTION
**Pull request by thomcsmits**

### CODE REVIEW

**Review by NickAkhmetov — COMMENTED** [review]
I'm a little bit out of my depth with distribution related questions, since I haven't maintained a public library like this before - @keller-mark and @manzt might have better input/more actionable advice since they have more experience with this matter.

I think the `umd` export might no longer be necessary since that's a somewhat old school format (e.g. react is dropping it) since it does add to the overall size of the package and is rarely necessary these days (it was used prior to the proliferation of ES modules as a universal adapter format for legacy back-end and front-end compatibility)

---

RE: `dist` files: it seems like most of the dist files are dependencies related to data loading and deserialization (e.g. lz4, jpeg, etc). These appear to be getting pulled in/built due to some deeply nested dependencies:

- `@vitessce/zarr` pulls in its `@vitessce/gl` dependency
- `@vitessce/gl` pulls in its `@hms-dbmi/viv` dependency
- `@hms-dbmi/viv` pulls in its `@vivjs/loaders` dependency
- `@vivjs/loaders` pulls in its `geotiff` dependency
- `geotiff` pulls in `lerc`, `pako`, and `zstddec`, which accounts for a few of the files in dist.

I suspect there's some similar dependency trees leading to the inclusion of `gzip`, `jpeg`, `lz4`, etc.

In conclusion, these dist files don't seem to be blockers, but do signal that there might be some improvements that could be made to tree-shaking (i.e. bundling settings might need to be adjusted to avoid pulling in these dependencies if they're not used on active code paths either here or further upstream in vitessce or viv).

After doing a bit more digging, I did find that there appears to be a related upstream `vite` issue that is caused by a limitation in `rollup`, so this could also just be out of our hands: https://github.com/vitejs/vite/discussions/13171

**Comment by manzt** [comment]
I'm in need of some context here. What is `dist`? Is it the library (for publishing to NPM) or the demo website?

**Comment by thomcsmits** [comment]
@manzt it's for loading the package into an anywidget with [jsdelivr](https://www.jsdelivr.com/?docs=gh). As this repo is not really ready for publishing to npm, but we would like to start creating the python implementation, I figured we could just pull it from gh, which I believe requires this dist.

**Comment by manzt** [comment]
I'd use a CDN that supports transpilation itself for this purpose, like esm.sh:

```
import("https://esm.sh/gh/hms-dbmi/cellpop@main/src/cellpop.ts")
```

**Comment by thomcsmits** [comment]
@manzt I see, thanks a lot!
